### PR TITLE
Update About Us 

### LINKED ISSUE #14: Update AboutUs - Rohan
Role - Testing
Responsibilities: Ensures the testing of the project is done properly and on time.

### PR DESCRIPTION
**Pull request by RSXIX**
Add Image to About Us

This commit introduces a new image to the About Us section of our website. The image is quite old because I am rushed for time.

"Closes #14"

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ### Rohan
 
-<img src="images/RSXIX.png" width="200px">
+<img src="images/rushedphotorohan.png" width="200px">
```
Comment (cyaoxuan) [inline docs/AboutUs.md]: The name of the file should be your Github username (see the [requirements](https://nus-cs2103-ay2324s1.github.io/website/schedule/week7/project.html#:~:text=The%20filename%20of%20the%20profile%20photo%20should%20be%20docs/images/github_username_in_lower_case.png))

File: docs/AboutUs.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 ## Project team
 
 ### Brendan
-
+git
```
Comment (cyaoxuan) [inline docs/AboutUs.md]: What is this?

**Review by cyaoxuan — CHANGES_REQUESTED** [review]

File: docs/AboutUs.md (hunk 1 above)
Comment (RSXIX) [reply docs/AboutUs.md]: Done

File: docs/AboutUs.md (hunk 2 above)
Comment (RSXIX) [reply docs/AboutUs.md]: Accident. Removed.

**Review by cyaoxuan — APPROVED** [review]
LGTM!

**Pull request merged**